6_visualize/in/6_outro_gif_tasks_template.mustache

```diff
@@ -23,7 +23,7 @@ targets:
       basemap_fun,
       ocean_name_fun,
       precip_raster_fun_a_{{last_task}},
-      storm_line_fun,
+      storm_line_fun_a_{{last_task}},
```

Comment:
wow, have we always used mustache templating for outro tasks? seems fine to me (and I like using whisker/mustache), just wondering why the deviation from a task table? This question is out of scope for this particular review, but I am curious nonetheless.

Reply:
This came along with the previous PR (#220) when I noticed we just had this file manually built. I didn't think this was a great candidate for task tables because none of the targets repeat. There are really only 3 output targets (3 frames) and they are all different, but share the value stored in `last_task`. Although, maybe I was approaching it wrong. I kept thinking that the "tasks" are the frame dates, but that is what worked in the storm gif tasks recipe. Maybe the "tasks" here are `rdgs`, `ltn`, and `readmore`.

Reply:
got it, good to know. I think I worked on these frames in the past but don't remember how we stitched them together.